tests/simpleKVBC/TesterReplica/setup.cpp

```diff
@@ -42,7 +41,7 @@ std::unique_ptr<TestSetup> TestSetup::ParseArgs(int argc, char** argv) {
   string idStr;
 
   int o = 0;
-  while ((o = getopt(argc, argv, "r:i:k:n:s:v:a:p")) != EOF) {
+  while ((o = getopt(argc, argv, "r:i:k:n:s:v:a:po:")) != EOF) {
```

Comment:
shouldn't this be a:p:o ?

Reply:
as per get getopt() specs, the option should have : after it only if it accepts value and here "p" is boolean option